Lib/multiprocessing/pool.py

```diff
@@ -173,7 +173,16 @@ def __init__(self, processes=None, initializer=None, initargs=(),
 
         self._processes = processes
         self._pool = []
-        self._repopulate_pool()
+        try:
+            self._repopulate_pool()
+        except OSError:
```

Comment:
Simply `except Exception` sounds ok to me.

Reply:
Sounds OK, it does not make sense to avoid leaking process only on known exceptions, and we're not hiding it.